modules/accredited_representative_portal/app/controllers/concerns/accredited_representative_portal/power_of_attorney_requests.rb

```diff
@@ -23,6 +23,13 @@ module PowerOfAttorneyRequests
             status: :unprocessable_entity
           )
         end
+
+        rescue_from Common::Exceptions::BadRequest do |e|
```

Comment:
This would need to handle `ActionController::BadRequest` now since we switched to raising that.